jetson/percep/main.cpp

```diff
@@ -24,8 +24,25 @@ bool cam_grab_succeed(Camera &cam, int & counter_fail) {
   return true;
 }
 
-static string rgb_foldername, depth_foldername;
```

Comment:
Please delete lines 27 through 70. We don't need the createVisualizerFunction() and the reset is just old commented code